ramls/types/assignedUsers/assignedUserDataAttributes.json

```diff
@@ -0,0 +1,45 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Assigned Users Collection Item Data Attributes Schema",
+  "description": "Assigned Users Collection Item Data Attributes Schema",
+  "javaType": "org.folio.rest.jaxrs.model.AssignedUserDataAttributes",
+  "additionalProperties": false,
+  "type": "object",
+  "properties": {
+    "credentialsId": {
+      "type": "string",
+      "description": "UUID of KB Credentials tied to",
+      "$ref": "../../raml-util/schemas/uuid.schema",
+      "examples": "2ffa1940-2cf6-48b1-8cc9-5e539c61d93f"
+    },
+    "userName": {
+      "type": "string",
+      "description": "A unique name belonging to a user",
+      "examples": "john_doe"
+    },
+    "firstName": {
+      "type": "string",
+      "description": "The first name of a user",
+      "examples": "John"
+    },
+    "middleName": {
+      "type": "string",
+      "description": "The middle name of a user (if any)",
+      "examples": "William"
+    },
+    "lastName": {
+      "type": "string",
+      "description": "The surname of a user",
+      "examples": "Doe"
+    },
+    "patronGroup": {
+      "type": "string",
+      "description": "The patron group name",
+      "examples": "Staff"
+    }
+  },
+  "required": [
+    "credentialsId",
```

Comment:
I think all fields except `middleName` are required because they will be used in the frontend.

Reply:
well, on user creation page 
<img width="1028" alt="Screenshot 2020-04-13 at 17 53 18" src="https://user-images.githubusercontent.com/37537790/79130654-fddba300-7daf-11ea-9fd8-b729588fbcd0.png">
there are several fields are required - Last name, patron group, status and email. A user can be created even without user name(but can not login into the system), middle name and first name.
so, the middle name is not required but thank you for review, because I found inaccuracies in JSON schemas.

Reply:
You add firstName but should lastName according to the info above

Reply:
Also db schema must be changed to have nullable columns for not required fields

Reply:
thanks, fixed